### DISCUSSION TITLE
How to handle module federated component notification plugin `_addNotification is not a function`

### DISCUSSION THREAD

**Discussion by iamprobirdey** [quasarframework/quasar#12563] [General - Components / Directives / etc]
Hey,

I am using **quasar** as a dependency on **Vue3**,

**Child app component is module federated**

On the **Parent** app I can get exposed component. :slight_smile: 

I expected to receive notification toast from the remote component where I used
`Notify` plugin from quasar.

**`_addNotification is not function error`**

**NB:** Notification plugins are used in both the apps and it's working totally fine.

**Comment by iamprobirdey** [comment]


**Reply by yusufkandemir** [reply]
I am not sure why you chose me. I also don't think specifically targeting somebody is a kind behavior.
If we let aside those, I don't think the question is clear. There is also no repro or code, so this is just guesswork. So, I don't know, and can't probably help unless you provide a repro.

**Reply by iamprobirdey** [reply]
Hey,
Apology for that. I didn't mean to disturb anyone in this manner.
